### DISCUSSION TITLE
Libre Computer boards

### DISCUSSION THREAD

**Discussion by dirkhh** [MichaIng/DietPi#6707] [Image Request]
I happen to have both a [Le Potato](https://libre.computer/products/aml-s905x-cc/) and a [Renegade](https://libre.computer/products/roc-rk3328-cc/). Both great little boards. Full open source drivers. Decent Armbian support.

I haven't looked into what it takes to add new boards at all...is there a howto? Or is it just "read the commits and figure it out"?

**Comment by Joulinar** [comment]
I guess for adding new boards we don't have a real how to. Best is to consult @MichaIng 🤣

**Comment by isarrider** [comment]
I happen to have a [La Frite](https://libre.computer/products/aml-s805x-ac/) lying around - think I backed it via kickstarter like 5 years ago...
Can give it a try also...

**Comment by 0jay** [comment]
I'd like a Renegade image also, I'd planned to use the LIbre portability tool but it's not been updated for Bookworm

**Comment by jtmoon79** [comment]
Relates to #5711 #6932
